build-deploy-platform-services/Jenkinsfile

```diff
@@ -58,6 +59,14 @@ node {
 		domain = params.domain.trim()
 	}
 
+	if (params.scm_branch == 'master') {
```

Comment:
My understanding of this story is , there is 13 platform services. By default it should be deploy in prod since UI is pointing to the prod API. Please correct if my understanding is wrong. I will add the null check